### ISSUE TITLE
向量余弦距离

### ISSUE THREAD

**Issue by zhaozanabc** [zzngb/autoqaFirst#1]
作者你好，我想问一下计算问题与正确、错误答案时候的余弦距离
with tf.name_scope("loss"):
    zero = tf.constant(0,  dtype=tf.float32)
    margin = tf.constant(0.05,  dtype=tf.float32)
    losses = tf.maximum(zero, tf.subtract(margin, tf.subtract(cos_12, cos_13)))
    losss = tf.reduce_sum(losses) + l2_reg_lambda * l2_loss
    print('loss ', losss)
这个原理是什么呢，将两个余弦值相减以后的操作是什么意思呢？谢谢

**Comment by zhaozanabc** [comment]
还有就是，那个所谓的错误答案是不是测试集啊，就是问题和正确答案是训练集，然后用“错误答案”检验训练出来的结果正确度怎么样？请教作者
